### PR TITLE
Fixing the StrictDate mapping to google.type.Date and …

### PR DESCRIPTION
**Pull request by itdnk**
…adding StrictTimestamp to be mapped to google.type.TimeOfDay.

### CODE REVIEW

**Comment by akphi** [comment]
@itdnk I don't think we should expose JIRA ticket number in the PR title, though nothing serious. If we have to associate these kinds of info, we probably should consider using [Github project with fields only available to us](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects#adding-metadata-to-your-tasks).

File: legend-engine-xt-protobuf-pure/src/main/resources/core_external_format_protobuf/transformation/fromPure/tests/pureToProtobufTests.pure [hunk 1]
```diff
@@ -126,18 +126,18 @@ function meta::external::format::protobuf::transformation::fromPure::tests::tran
...
                 '}'
                 , $resultForTrade.second);
-  let resultForAccount = $results->filter( r | $r.first == 'meta_external_format_protobuf_transformation_fromPure_tests_account.proto');              
+  let resultForAccount = $results->filter( r | $r.first == 'meta_external_format_protobuf_transformation_fromPure_tests_account.proto');
   assertEquals('syntax = "proto3";\n' +
                 'package meta.external.format.protobuf.transformation.fromPure.tests.account;\n' +
                 '\n' +
-                'import "google/protobuf/timestamp.proto";\n' +
-                'import "meta_external_format_protobuf_transformation_fromPure_tests_trade.proto";\n' +
+                'import "google/type/date.proto";\n' +
```
Comment (rafaelbey) [inline legend-engine-xt-protobuf-pure/src/main/resources/core_external_format_protobuf/transformation/fromPure/tests/pureToProtobufTests.pure:133]: Can we add coverage for all the defined types (Date/DateTime/StrictTime)?  With the proposed changes, seems we only cover StrictDate.

Also, it seems we don't execute pure test cases for this module (given the assertion changes)  Is this a miss on the module setup or a explicit choice?

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**